examples/data_loss_prevention/dlp_stages/dlp_input_processor.py

```diff
@@ -0,0 +1,117 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import time
+import typing
+
+import mrc
+from mrc.core import operators as ops
+
+from morpheus.cli.register_stage import register_stage
+from morpheus.config import Config
+from morpheus.config import PipelineModes
+from morpheus.messages import ControlMessage
+from morpheus.messages import MessageMeta
+from morpheus.pipeline.control_message_stage import ControlMessageStage
+from morpheus.pipeline.execution_mode_mixins import GpuAndCpuMixin
+from morpheus.utils.type_aliases import SeriesType
+from morpheus.utils.type_utils import get_df_class
+
+logger = logging.getLogger(f"morpheus.{__name__}")
+
+
+@register_stage("dlp_input_processor", modes=[PipelineModes.NLP])
+class DLPInputProcessor(GpuAndCpuMixin, ControlMessageStage):
+    """
+    Handles input text processing and normalization for DLP pipeline
+
+    Parameters
+    ----------
+    config : morpheus.config.Config
+        Pipeline configuration instance.
+    column_name : str
+        Name of the column containing the source text to process.
+    split_paragraphs : bool
+        If True, splits input text into chunks. Defaults to False.
```

Comment:
Should we specify the default values if unspecified?

`log_elapsed_time` is also missing.